graphics/src/STLLoader.cc

```diff
@@ -28,9 +28,16 @@
 using namespace ignition;
 using namespace common;
 
+
+//////////////////////////////////////////////////
+class ignition::common::STLLoader::Implementation
+{
+};
+
```

Comment:
Since this one is empty was this some sort of scripting/automation thing?

Reply:
Not really I just went and added implementation pointers to all the classes, the main idea was that if in the future we had to add any data member it wouldn't cause API breakage but I'm not sure how likely the case is for the `STLLoader` class

Reply:
Ah this makes more sense, good call. May not be universally useful, but does potentially save some headaches later.